docs/api/registration-base-url-resource.md

```diff
@@ -216,6 +217,26 @@ registration | string | no       | The URL to the registration index for this de
 If the `range` property is excluded or an empty string, the client should default to the version range `(, )`. That is,
 any version of the dependency is allowed.
 
+#### Package deprecation
+
+Each package deprecation has the following properties:
+
+Name             | Type             | Required | Notes
+---------------- | ---------------- | -------- | -----
+reasons          | array of strings | yes      | The reasons why the package was deprecated
```

Comment:
Is empty array a valid value?

Reply:
It's unclear that you can't put whatever values you'd like here. I would consider listing the allowed reasons here.
```suggestion
reasons          | array of strings | yes      | The package deprecation reason. Allowed values are `Legacy`, `CriticalBugs`, or `Other`.
```

Reply:
Will specify that an empty array is not valid.

As much as I would like to add these details in the notes column, I think it would be too verbose for the table. Furthermore, the rest of the documentation follows the pattern of providing more details about the properties below the property table (see `Package dependency` and `Package dependency group` above).